README.md

```diff
@@ -148,14 +187,15 @@ Write ruby code that will create a person.
 
 Your answer:
 ```
-Replace this with your answer
+INSERT INTO persons (name, age)
+VALUES ('Ba Phan', 24)
 ```
 
 Write ruby code that will query for any person that is 15 years of age
 
 Your answer:
 ```
-Replace this with your answer
+SELECT * FROM persons WHERE age =< 15
 ```
 
```

Comment:
SQL needs semicolon.

what ruby/active record method would we use, and with what params?